src/jobs/BenchmarkJob.jl

```diff
@@ -315,7 +315,7 @@ function execute_benchmarks!(job::BenchmarkJob, whichbuild::Symbol)
                       # move ourselves onto the first CPU in the shielded set
                       run(`sudo cset proc -m -p \$(getpid()) -t /user/child`)
 
-                      blas_set_num_threads(1) # ensure BLAS threads do not trample each other
+                      $(VERSION < v"0.5.0-dev+4338" ? blas_set_num_threads(1) : BLAS.set_num_threads(1)) # ensure BLAS threads do not trample each other
```

Comment:
I think interpolating it is actually incorrect, since it would use the server's `VERSION` rather than the build's `VERSION`. So I think what we need is the non-interpolated version. Thanks again!